dapps/tests/app/test/simple_storage_deploy_spec.js

```diff
@@ -5,12 +5,9 @@ const {Utils} = require('Embark/EmbarkJS');
 contract("SimpleStorage Deploy", function () {
   let simpleStorageInstance;
   before(function(done) {
-    Utils.secureSend(web3, SimpleStorage.deploy({arguments: [150]}), {}, true, function(err, receipt) {
-      if(err) {
-        return done(err);
-      }
-      simpleStorageInstance = SimpleStorage;
-      simpleStorageInstance.options.address = receipt.contractAddress;
+    SimpleStorage.deploy([150]);
+    SimpleStorage.deployed().then(instance => {
```

Comment:
Could we also add the promise resolution to the `deploy()` function to these test cases, maybe as a separate `describe`, but with the same unit tests? For example,
```
contract("SimpleStorage Deploy Promise", function () {
  let simpleStorageInstance;	
  SimpleStorage.deploy([150]).then(instance => {
      simpleStorageInstance = instance;
      simpleStorageInstance.options.address = receipt.contractAddress;	
      done();
    });	
    // same it blocks go here
```